racunaljka.js

```diff
@@ -0,0 +1,68 @@
+var writes = document.getElementById('screen');
```

Comment:
You are going in the right direction, but still using too much of string manipulation. Focus on numbers, not strings. Here is a short hint to push you toward optimal solution:
```
var currNumber = 0;
var result = 0;

...

numberButton[i].onclick = function() {

  var buttonNumber = parseInt(this.innerHtml);
  currNumber = currNumber * 10 + buttonNumber;

}

...

sumButton.onclick = function() {

  result += currNumber;
  currNumber = 0;
  screen.value = result;

}
```